docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 server:
   build: .
   volumes:
-    - .:/app
-    - /app/node_modules
+    - ./conf.js:/app/conf.js
```

Comment:
now that i think of it maybe `docker-compose.development.yml` makes more sense, as long as there is a flag for --paper trading by default in both `Dockerfile` and `docker-compose`